docs/policy-only.md

```diff
@@ -0,0 +1,156 @@
+routed by it.
+
+When a Pod is instantiated, the container runtime first calls the primary CNI to configure
+Pod's IP, route table, DNS etc, and then connects Pod to host network with a PtP device such as a
+veth-pair. When CNI chaining is configured, container runtime then calls Antrea, and the Antrea
+agent attaches Pod's PtP device to the OVS bridge, and moves the host route to the Pod to local
+host gateway(``gw0``) interface from the PtP device. This is illustrated by the diagram on the
+right.
+
+Antrea needs to satisfy that 
+1. All IP packets, sent on ``gw0`` in the host network, are received by the Pods exactly the same
+as if the OVS bridge has not been inserted. 
+1. Similarly all IP packets, sent by Pods, are received by other Pods or the host network exactly
+the same as if OVS bridge has not been inserted.
+1. There are no requirements on Pod MAC addresses as all MAC addresses stays within the OVS bridge.
+
+To satisfy the above requirements, Antrea needs no knowledge of Pod's network configurations nor
+nor underlying CNI network, it simply needs to program the following OVS flows on the OVS bridge:
+1. A default ARP responder flow that answers any ARP request. Its sole purpose is so that a Pod's
+neighbor may be resolved, and packets may be sent by that Pod to that neighbor.
+1. IP packets are routed based on its destination IP if it matches any local Pod's IP.
+1. All other IP packets are routed to host network via gw0 interface.
+
+These flows together handle all Pod traffic patterns with exception of Pod-to-Service traffic
+that we will address next.
+
+## Handling Pod-To-Service
+The discussion in this section is relevant also to Pod-to-Service traffic in NoEncap traffic
+mode. Antrea applies same principle to handle Pod-to-service traffic in all traffic modes where
+traffic requires no encapsulation.
+
+Antrea uses kube-proxy for load balancing. At the same time, it supports also Pod level
+NetworkPolicy enforcement.
+
+This means that for any Pod-to-Service traffic flow, it needs to  
+1. first traverse to the host network for load balancing (DNAT), then
+1. comes back to OVS bridge for Pod NetworkPolicy processing, and
+1. if DNATed destination in 1) is an inter-Node Pod or external network entity, and goes back
+to the host network yet again to be forwarded.
+
+We refer the last traffic pattern as re-entrance traffic because in this pattern, an traffic flow
+enters host network twice -- first time for load balancing, and second time for forwarding.
+
+Specifically a re-entrance traffic may be traced as follows:<br>
+Denote VIP as cluster IP of a service, SP_IP/DP_IP as respective client and server Pod IP; 
+VPort as service port of a service, TPort as target port of server Pod, and SPort as original 
+source port. The service request's 5-tuples upon first and second entrance to the host network, and
+its reply's 5-tuples would be like
```

Comment:
IMO, this would benefit from being presented as a list

Reply:
can u give a visual example with list ?

Reply:
done.

Reply:
hmm still think the above should be a list:
* `VIP` is the ClusterIP of a service
* `SP_IP` is the IP address of the client Pod
...

I wasn't referring to what came below in my original comment

Reply:
done.